### ISSUE TITLE
Need land files for resolutions besides 120km

### ISSUE THREAD

**Issue by gdicker1** [EarthWorksOrg/EarthWorks#74]
As reported by @dazlich, runs fail at resolutions higher than mpasa120 since updating EarthWorksOrg/CTSM (clm) to be based on ctsm5.2.X.

At least the fsurdat files need to be regenerated; the ctsm5.1 files are missing a `pct_ocean` variable.

**Comment by gdicker1** [comment]
From looking at the [current namelist_defaults_ctsm.xml](https://github.com/EarthWorksOrg/CTSM/blob/72e4235075e1b5c98d5cfe68173ac35f2edf93f6/bld/namelist_files/namelist_defaults_ctsm.xml) there are some files for 60, 15, and 3.75km MPAS resolutions. However, I don't know if it's important that these files only apply with `use_crop=".false."

```
<fsurdat hgrid="mpasa60"   sim_year="2000" use_crop=".false.">
lnd/clm2/surfdata_esmf/ctsm5.2.0/surfdata_mpasa60_hist_2000_16pfts_c240216.nc</fsurdat>
<fsurdat hgrid="mpasa15"   sim_year="2000" use_crop=".false.">
lnd/clm2/surfdata_esmf/ctsm5.2.0/surfdata_mpasa15_hist_2000_16pfts_c240216.nc</fsurdat>
<!--
<fsurdat hgrid="mpasa15-3" sim_year="2000" use_crop=".false.">
lnd/clm2/surfdata_esmf/ctsm5.2.0/surfdata_mpasa15-3_hist_2000_16pfts_c240216.nc</fsurdat>
-->
<fsurdat hgrid="mpasa3p75" sim_year="2000" use_crop=".false.">
lnd/clm2/surfdata_esmf/ctsm5.2.0/surfdata_mpasa3p75_hist_2000_16pfts_c240216.nc</fsurdat>
```

Later in the file we see the ctsm5.2 fsurdat files for mpasa480 and mpasa120

```
<fsurdat hgrid="mpasa480"  sim_year="2000">
lnd/clm2/surfdata_esmf/ctsm5.2.0/surfdata_mpasa480_hist_2000_78pfts_c240216.nc</fsurdat>
<fsurdat hgrid="mpasa120"  sim_year="2000">
lnd/clm2/surfdata_esmf/ctsm5.2.0/surfdata_mpasa120_hist_2000_78pfts_c240216.nc</fsurdat>
```

Searching this file for other mentions of '`hgrid="mpasa`', we may have to also look at files for `flanduse_timeseries`. There is currently only files for mpasa120 resolution:

```
<flanduse_timeseries hgrid="mpasa120"      sim_year_range="1850-2000"
>lnd/clm2/surfdata_esmf/ctsm5.2.0/landuse.timeseries_mpasa120_SSP2-4.5_1850-2100_78pfts_c240216.nc</flanduse_timeseries>
``` 

AND

```
<flanduse_timeseries hgrid="mpasa120"      sim_year_range="1850-2100" ssp_rcp="SSP2-4.5"
>lnd/clm2/surfdata_esmf/ctsm5.2.0/landuse.timeseries_mpasa120_SSP2-4.5_1850-2100_78pfts_c240216.nc</flanduse_timeseries>
```

**Comment by gdicker1** [comment]
The PR to create CTSM files for EarthWorks may be paused for a bit. It looks like there's some issues with the `mksurfdata_esmf` tool not filling in the data for soil categories in some regions. This may be fixed soon when CTSM/5.3 comes out and is incorporated into CESM (may need to update the EarthWorks tags again to allow this fix).

See: https://github.com/ESCOMP/CTSM/issues/2744

@dazlich @sherimickelson @areanddee, what are your thoughts? Would it be better to get things functional sooner (i.e. go ahead and create some CTSM files) or to wait for the CTSM tool update?

**Comment by gdicker1** [comment]
Given discussion in the regular EarthWorks SEs meeting today, I will move this forward. It will be better to be functional sooner.

**Comment by gdicker1** [comment]
Resolved by merge of #73. Please revive this issue or start a new one if problems are noticed.
